### ISSUE TITLE
Tree table is causing the a11y test to fail

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#3861]
We added some `aria` attrs to support tree view (following the [w3 tree grid example](https://www.w3.org/TR/wai-aria-practices/examples/treegrid/treegrid-1.html)), and the a11y tests are failing. They fail on the w3 example, too.
